src/zzz_od/auto_battle/auto_battle_context.py

```
@@ -606,6 +609,16 @@ def check_battle_state(
             for future in future_list:
                 future.result()
 
+        # debug 闪避检测截图
+        if self.ctx.env_config.dodge_detect and flash_future is not None and audio_future is not None:
+            try:
+                flash_detected = flash_future.result()
+                audio_detected = audio_future.result()
+                if flash_detected or audio_detected:
+                    debug_utils.save_debug_image(screen, prefix='dodge')
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)